.travis.yml

```diff
@@ -8,18 +8,12 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.6.*
+  - SYMFONY_VERSION=2.7.*
```

Comment:
you should have a build not specifying any Symfony version but running with unmodified deps IMO. Otherwise, you have no job showing that your requirements are defined properly (and you should also have a job testing lowest deps IMO to check the lowest bound)